hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeIntegrationTest.java

```diff
@@ -163,5 +167,75 @@ public void testCacheMetrics() {
                 .withQueryParam("cawact", equalTo("1")));
     }
 
+    @Test
+    public void testMapLatencies() {
+        DelayMapStore mapStore = new DelayMapStore();
+        MapStoreConfig mapStoreConfig = new MapStoreConfig();
+        mapStoreConfig.setEnabled(true);
+        mapStoreConfig.setClassName(getClass().getName());
+        mapStoreConfig.setImplementation(mapStore);
+        IMap<String, String> iMap = node.hazelcastInstance.getMap("hazelcast");
+        node.getConfig().getMapConfig("hazelcast").setMapStoreConfig(mapStoreConfig);
+        iMap.put("1", "hazelcast");
+
+
+        stubFor(get(urlPathEqualTo("/ping"))
+                .willReturn(aResponse()
+                        .withStatus(200)));
+
+        phoneHome.phoneHome(false);
+        verify(1, getRequestedFor(urlPathEqualTo("/ping"))
+                .withQueryParam("mpptla", equalTo(String.valueOf(iMap.getLocalMapStats().getTotalPutLatency() / iMap.getLocalMapStats().getPutOperationCount())))
+                .withQueryParam("mpgtla", equalTo("-1")));
+
+        assertGreaterOrEquals("mpptla", iMap.getLocalMapStats().getTotalPutLatency() / iMap.getLocalMapStats().getPutOperationCount(), 200);
```

Comment:
So what happens is that first in an equalTo() call checks if the actual value is equal to a computed value, then a second assertion checks that the computed value is approximately correct.

It would be better to put a greaterThanOrEqualTo() call into the wiremock verification call itself, but that doesn't seem to be supported by wiremock. Briefly looking at wiremock matchers, you can create a custom `GreaterThanOrEualToPattern` class which extends `StringValuePattern` and override the `match()` method. Then you can use this newly created matcher like `.withQueryParam("mpptla", new GreaterThanOrEqualToPattern(200))`

Reply:
Okay I'll look into this.